### PR TITLE
Add sql subcommand for gitbase mysql client

### PR DESCRIPTION
**Pull request by smacker**
Signed-off-by: Maxim Sukharev <max@smacker.ru>

### CODE REVIEW

**Review by carlosms — CHANGES_REQUESTED** [review]
Missing docs, entry in changelog

File: cmd/sourced/cmd/sql.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/src-d/sourced-ce/cmd/sourced/compose"
+)
+
+type sqlCmd struct {
+	Command `name:"sql" short-description:"Open a mysql client connected for gitbase" long-description:"Open a mysql client connected for gitbase"`
```
Comment (carlosms) [inline cmd/sourced/cmd/sql.go]:
```suggestion
	Command `name:"sql" short-description:"Open a MySQL client connected to gitbase" long-description:"Open a MySQL client connected to gitbase"`
```

**Comment by carlosms** [comment]
Will it work on windows?

**Comment by smacker** [comment]
Thanks! Re-pushed with updated documentation.

It should work on windows. (at least `docker exec -it` worked as expected) I can test tomorrow.

**Comment by smacker** [comment]
works as a charm (srcd was inited without repos):
Screenshot 2019-06-13 at 11 52 15

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ The `sourced` binary is a wrapper for Docker Compose that downloads the `docker-
 - `stop`: Stop running containers
 - `start`: Start stopped containers
 - `web`: Open the web interface in your browser
+- `sql`: Open a MySQL client connected for gitbase
```
Comment (carlosms) [inline CHANGELOG.md]:
```suggestion
- `sql`: Open a MySQL client connected to gitbase
```

File: README.md [hunk 3]
```diff
@@ -127,6 +127,14 @@ sourced web
 
 Opens the web interface in your browser.
 
+#### SQL
+
+```
+sourced sql
+```
+
+Open a MySQL client connected for gitbase.
```
Comment (carlosms) [inline README.md]:
```suggestion
Open a MySQL client connected to gitbase.
```

File: cmd/sourced/cmd/sql.go [hunk 4]
```diff
@@ -0,0 +1,19 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/src-d/sourced-ce/cmd/sourced/compose"
+)
+
+type sqlCmd struct {
+	Command `name:"sql" short-description:"Open a MySQL client connected for gitbase" long-description:"Open a MySQL client connected for gitbase"`
```
Comment (carlosms) [inline cmd/sourced/cmd/sql.go]:
```suggestion
	Command `name:"sql" short-description:"Open a MySQL client connected to gitbase" long-description:"Open a MySQL client connected to gitbase"`
```

**Review by carlosms — APPROVED** [review]
:+1: after the for/to text change

**Comment by smacker** [comment]
for/to changed

**Pull request merged**